src/simulator/interpret.cpp

```diff
@@ -1109,23 +1111,52 @@ namespace CoreIR {
       updateLUTNNode(vd);
     } else if (opName == "float.add") {
       updateBitVecBinop(vd, [](const BitVec& l, const BitVec& r) {
-          assert(l.bitLength() == 32);
-          assert(r.bitLength() == 32);
 
-          int lv = l.to_type<int>();
-          int rv = r.to_type<int>();
+          if ((l.bitLength() == 32) && (r.bitLength() == 32)) {
+            int lv = l.to_type<int>();
+            int rv = r.to_type<int>();
 
-          float lf = bitCastToFloat(lv);
-          float rf = bitCastToFloat(rv);
+            float lf = bitCastToFloat(lv);
+            float rf = bitCastToFloat(rv);
 
-          cout << "lf = " << lf << endl;
-          cout << "rf = " << rf << endl;
+            cout << "lf = " << lf << endl;
```

Comment:
Looks like there are some debug statements here and elsewhere in this file. Could you remove them?